src/main/java/frc/robot/commands/KsCharacterization.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.SwerveSubsystem;
+import frc.robot.swerve.SwerveModuleAbstract;
+
+public class KsCharacterization extends Command {
+  /** Creates a new kS_Characterization. */
+  private final SwerveSubsystem mSwerve;
+  private final SwerveModuleAbstract[] mModules;
+  private double [] expkS = new double[4];
+  private boolean [] modLock = {false,false,false,false};
+  private double curRequestedDutyCycle = 0;
+  public KsCharacterization(SwerveSubsystem swerve) {
```

Comment:
Add a blank line before the function declaration.

Reply:
done